IdentityCore/tests/MSIDAADRequestConfiguratorTests.m

```diff
@@ -76,7 +77,7 @@ - (void)testConfigure_shouldConfigureAADRequest
     XCTAssertNotNil(headers[@"x-client-OS"]);
     XCTAssertNotNil(headers[@"x-client-SKU"]);
     XCTAssertNotNil(headers[@"x-client-Ver"]);
-    XCTAssertNotNil(headers[@"x-ms-PkeyAuth"]);
+    XCTAssertNotNil(headers[kMSIDPKeyAuthHeader]);
```

Comment:
Where do you update the PkeyAuth+ header that we send now? I couldn't find that code

Reply:
kMSIDPKeyAuthHeader is defined as 'x-ms-PKeyAuth+' in IdentityCore/src/workplacejoin/MSIDWorkPlaceJoinConstants.m

Reply:
Discussed offline, resolving